src/app/docs/docs.html

```diff
@@ -247,11 +246,224 @@ <h3>Badge on Icons</h3>
             </figure>
           </script>
         </div>
-
-        <hr />
       </section>
 
       <!-- Badge ends -->
+
+      <hr />
+
+      <!-- Alerts starts-->
+
+      <section class="components" id="alert">
+        <h2 class="component__heading">Alerts</h2>
+        <p class="component__sub-heading">
+          Alert messages can be used to notify the user about something special:
+          danger, success, information or warning without interrupting the
+          user's flow.
+        </p>
+        <div class="components__parts components--flex--column">
+          <div class="alert alert--primary" role="alert">
+            A primary alert with text link <a href="#">Check it out!</a>
+          </div>
+          <div class="alert alert--secondary" role="alert">
+            A secondary alert
+          </div>
+          <div class="alert alert--success" role="alert">
+            <span><i class="far fa-check-circle"></i></span>A success alert
+          </div>
+          <div class="alert alert--danger" role="alert">
+            <span><i class="fas fa-exclamation-triangle"></i></span>A danger
+            alert
+          </div>
+          <div class="alert alert--warning" role="alert">
+            <span><i class="fas fa-exclamation-circle"></i></span>A warning
+            alert
+          </div>
+          <div class="alert alert--info" role="alert">
+            <span><i class="fas fa-info-circle"></i></span>A info alert
+          </div>
+        </div>
+        <p>
+          Alerts are available for any length of text. For proper styling, use
+          one of the seven required contextual classes
+        </p>
+        <span>
+          <span><b>alert--primary</b></span>
+          <span><b>alert--secondary </b></span>
+          <span><b>alert--success </b></span>
+          <span><b>alert--danger </b></span>
+          <span><b>alert--warning </b></span>
+          <span><b>alert--info </b></span>
+        </span>
+
+        <div class="code-Snippet">
+          <script
+            type="text/plain"
+            class="line-numbers language-css snippet_container__script"
+          >
+            <!-- Primary alert -->
+            <div class="alert alert--primary " role="alert">
+               A primary alert with text link <a href="#">Check it out!</a>
+             </div>
+
+             <!-- Secondary alert -->
+             <div class="alert alert--secondary" role="alert">A secondary alert</div>
+
+             <!-- Success alert -->
+             <div class="alert alert--success">
+               <span><i class="far fa-check-circle"></i></span>A success alert
+             </div>
+
+             <!-- Danger alert -->
+             <div class="alert alert--danger" role="alert">
+               <span><i class="fas fa-exclamation-triangle"></i></span>A danger
+               alert
+             </div>
+
+             <!-- Warning alert -->
+             <div class="alert alert--warning" role="alert">
```

Comment:
Great work. You wrote the role attribute which helps screen readers to read this element as defined.

Reply:
Thanks Rahul